### PR TITLE
Add CI.

### PR DESCRIPTION
**Pull request by vext01**
One of the test suites doesn't seem to work. Have asked on discord, but I don't think this should block us.

### CODE REVIEW

**Comment by vext01** [comment]
bors ping

File: .buildbot.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+# https://llvm.org/docs/TestingGuide.html
+#
+# This runs unit and integration tests.
+make -j `nproc` check-all
+cd ..
+
+# This runs the `test-suite` tests.
+# https://llvm.org/docs/TestSuiteGuide.html
+#
+# FIXME But it doesn't seem to work:
```
Comment (ltratt) [inline .buildbot.sh:27]: I'm not sure what this comment (and commented out code) mean.

Comment (vext01) [reply .buildbot.sh:27]: There are three test suites and one is, confusingly, called `test-suite`. The commented code would run this suite, but it fails.

See  https://llvm.org/docs/TestingGuide.html

Comment (ltratt) [reply .buildbot.sh:27]: Why does it fail?

Comment (vext01) [reply .buildbot.sh:27]: The error message is shown in the comment. I've asked on discord, but not heard back.

I've tried the master branch, the llvmorg-12.0.0 tag and the release/12.x branch :\

Comment (ltratt) [reply .buildbot.sh:27]: OK, can we rephrase this to be clearer? Something like "The shell code below should run the `test-suite` tests but these currently fail with the following error: XXX".

Comment (vext01) [reply .buildbot.sh]: Try 92a709d8d

**Comment by ltratt** [comment]
Please squash.

**Comment by vext01** [comment]
splat.

**Comment by vext01** [comment]
bors try

**Comment by vext01** [comment]
bors try-

**Comment by vext01** [comment]
bors try

**Comment by vext01** [comment]
bors try-

The build succeeded, but the bors.toml was wrong (I've renamed all references to "Yorick" to "yk").

I'll raise a PR for the bb config, then we can squash and merge this.

**Comment by vext01** [comment]
bors try

**Comment by vext01** [comment]
bors try

**Comment by vext01** [comment]
bors try-

**Comment by vext01** [comment]
bors try

**Comment by vext01** [comment]
I think I've solved the CI issue.

OK to squash?

**Comment by ltratt** [comment]
Please squash.

**Comment by vext01** [comment]
splat.
